projects/planner/src/app/diver/diver.component.html

```diff
@@ -11,15 +11,15 @@
             <label for="stressRmv" class="form-label" mdbLabel>Stress RMV [{{units.rmv}}]:</label>
             <input class="form-control" id="stressRmv" formControlName="stressRmv" type="number" (input)="inputChanged()"
                    [min]="ranges.diverRmv[0]" [max]="ranges.diverRmv[1]" [step]="rmvStep"
-                   [class.is-invalid]="stressRmvInvalid" />
+                   [class.is-invalid]="stressRmvInvalid" id="stressRmv" />
             <div class="invalid-feedback position-absolute">Needs to be number {{ranges.diverRmvLabel}}</div>
         </div>
         <div class="pt-2 mt-2 mb-4 col col-12 col-md-6">
-            <app-pp-o2 [maxPpO2]="diver.maxPpO2" (ppO2Change)="maxPpO2Changed($event)" label="Maximum ppO2:"
+            <app-pp-o2 id="maxPpO2" [maxPpO2]="diver.maxPpO2" (ppO2Change)="maxPpO2Changed($event)" label="Maximum ppO2:"
```

Comment:
app-pp-o2 is custom element, which internally contains input html element, you should refence its ID.